stat_key_browser/cluster_config.py

```diff
@@ -5,10 +5,10 @@
     try:
         # Try the original SDK library name
         import isi_sdk
```

Comment:
Seems like this should be:
```python
import isi_sdk_7_2
```
or
```python
import isi_sdk_7_2 as isi_sdk
```

Reply:
The intent here is to allow functioning with either `isi_sdk_7_2` or the original and now deprecated `isi_sdk` library. 

So it first tries importing isi_sdk_7_2, if that fails then it tries for the deprecated SDK library.